microdnf/context.cpp

```diff
@@ -306,7 +328,7 @@ void Context::load_rpm_repos(libdnf::repo::RepoQuery & repos, libdnf::rpm::Packa
         }
     };
 
-    std::cout << "Updating repositories metadata and load them:" << std::endl;
+    print_info("Updating repositories metadata and load them:", get_quiet());
```

Comment:
If you made `print_info()` a member of `Context`, you wouldn't have to pass it the `quiet` argument, it could call `get_quiet()` from inside, couldn't it?

Reply:
Done